huggan/tensorflow/dcgan/README.md

```diff
@@ -0,0 +1,50 @@
+## Train DCGAN on your custom data
+This folder contains a script to train DCGAN for unconditional image generation, leveraging the Hugging Face ecosystem for processing your data and pushing the model to the Hub.
+
+The script leverages 🤗 Datasets for loading and processing data, and TensorFlow for training the model and 🤗 Hub for hosting it.
```

Comment:
```suggestion
The script leverages 🤗 Datasets for loading and processing data, TensorFlow for training the model and 🤗 Hub for hosting it.
```

Reply:
Actually grammar-wise the current one is already correct I think :smile:

Reply:
Haha yeah sorry this is a major nitpick. I usually only include "and" at the end